OrbitBase/include/OrbitBase/ThreadConstants.h

```diff
@@ -9,5 +9,12 @@ namespace orbit_base {
 // Represents a fake thread id to specify the set of all thread ids of the current process.
 static constexpr int32_t kAllProcessThreadsFakeTid = -1;
 
+// Represents a fake thread id to specify the set of all thread ids of all processes on the system.
+static constexpr int32_t kAllThreadsOfAllProcessesFakeTid = -2;
```

Comment:
nit: I would drop Fake from the name, it does not add anything.

Reply:
Agree, thanks!